docs/linux_dependencies.md

```diff
@@ -136,15 +136,15 @@ You can do this in one line with `nix-shell --run "cargo run"`.
 
 This is also possible with [Nix flakes](https://nixos.org/manual/nix/unstable/command-ref/new-cli/nix3-flake.html).
 Instead of creating `shell.nix`, you just need to add the derivation (`mkShell`)
-to your `devShells` in `flake.nix`. Run `nix develop` to enter the shell and
+to your `devShells` in `flake.nix`. Run `nix develop` to enter the shell or just
```

Comment:
"or just" (longer command) sounds a bit wrong.

If we want "just" here, it should probably be later. "(command) to just run the program."